src/osd/OSD.h

```diff
@@ -563,7 +563,7 @@ class OSDService : public Scrub::ScrubSchedListener {
 				   unsigned int qu_priority,
 				   Scrub::act_token_t act_token);
 
-  void queue_for_pg_delete(spg_t pgid, epoch_t e);
+  void queue_for_pg_delete(spg_t pgid, epoch_t e, int64_t num_objects=1);
```

Comment:
Why do we need a default here?  Which caller wants to pass 1?

Reply:
I wasn't sure how to handle queue_for_pg_delete here: https://github.com/ceph/ceph/blob/main/src/osd/PG.cc#L2668. It will only be called with osd_op_queue=WPQ since osd_delete_sleep has to be > 0.

Reply:
Pass 1 explicitly with a comment:

```/* We pass 1 for num_objects here as it only wpq uses this code path and it will be ignored */```

Reply:
No, I mean at the single actual call site -- remove the default.  I don't want users to accidentally leave off the argument.